examples/grpc/src/main/java/io/opentelemetry/example/HelloWorldServer.java

```diff
@@ -91,7 +91,7 @@ private void stop() {
 
   private void initTracer() {
     // Get the tracer management instance
-    TracerSdkManagement tracerManagement = OpenTelemetrySdk.getTracerManagement();
+    TracerSdkManagement tracerManagement = OpenTelemetrySdk.getGlobalTracerManagement();
```

Comment:
we shouldn't update examples until we do a release, correct? They should be pointing at the latest release, not a snapshot.